app.js

```diff
@@ -10,16 +9,48 @@ const punkAPI = new PunkAPIWrapper();
 app.set('view engine', 'hbs');
 app.set('views', path.join(__dirname, 'views'));
 
+
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
-
-// ...
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+
+  const allBeers = punkAPI.getBeers()
+  allBeers.then(beersFromApi => {
+    res.render('beers-page', { beerObject: beersFromApi })
+  // console.log(allBeers)
+    console.log('Beers from the database: ', beersFromApi)
+  })
+  allBeers.catch(error => console.log(error));
+ 
+  
+});
```

Comment:
No es necesario guardarnos la petición en una constante, en adelante lo evitaremos, simplemente porque es un paso inocuo.

```suggestion
app.get('/beers', (req, res) => {


  punkApi.getBeers().then(beersFromApi => {
    res.render('beers-page', { beerObject: beersFromApi })
  // console.log(allBeers)
    console.log('Beers from the database: ', beersFromApi)
  })
  allBeers.catch(error => console.log(error));
 
  
});
```